examples/qasm.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
<blockquote> QuTiP also produces custom gate definitions for gates not provided in the <code>qelib1.inc</code> "header" file.</blockquote><p>Do we support exporting QuTiP customized gates or not? Could you make it a bit more clearer in the text?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>